docs/home/pwa-intro.md

```diff
@@ -0,0 +1,150 @@
+
+!> Service workers only work with applications that are secured through `HTTPS`. Before you can add a service worker, make sure your app is secure.
+
+## Installability
+
+A web app that has a manifest and a service worker can be locally installed! *Installability* is one of the key lines that separates classic web apps and their progressive counterparts.
+
+Just like a native application, an installed PWA can be launched from the operating system, pinned to desktops and taskbars, and indexed by your device's search software.
+
+#### Installing from the Browser
+
+PWAs can be installed directly from both Google Chrome and Microsoft Edge.
+
+Both browsers have an install button at the right end of the address bar that will appear if the current page is a PWA. 
+
+Here's what the button and follow-up prompt look like for a PWA in Edge:
+
+<div class="docs-image">
+   <img src="assets/home/beginners/install-prompt.png" alt="An example PWA with the browser install prompt open" width=600>
+</div>
+
+Clicking install is all you need to access your PWA from the OS!
+
+#### Installing from App Stores
+
+Depending on the platform, PWAs are now able to be installed from various app stores.
+
+The Microsoft, Google Play, and Meta Quest stores all natively support PWAs, and the install process is the same as it would be for any app. Just search for it in the store of choice and click install.
+
+Other platforms, such as iOS, don't natively support PWAs, but PWAs can sometimes be wrapped in native frameworks to allow publishing to stores.
+
+## Design
+
+Adding a manifest and service worker to your PWA and making it installable are just the tip of the iceberg for building quality progressive web apps.
+
+PWAs are different then websites, and have a different standard when it comes to user experience.
+
+A well-designed PWA will check a handful of boxes:
+
+- Smooth, modern, and consistent UI and navigation experience (for example, the view shouldn't refresh on navigation)
+
+- Reliable offline experience. Even if your app is heavily network dependent, it should remain basically responsive when offline.
+
+- Takes advantage (when appropriate) of modern web capabilities to deliver an integrated application experience. 
+
+
+In general, a good rule of thumb is that a progressive web app should feel more like an application than a website.
+
+
+## Demo
+
+This documentation is a PWA itself, and can be installed and used to explore some basic PWA features.
+
+!> Make sure you are using Edge or Chrome before trying to install the PWA.
+
+Here are some things to try:
+
+
+#### Install this PWA
+
+Using the guidance from the *Installability* section, install this PWA. The button is located on the far right of the address bar in either Chrome or Edge.
+
+Once the PWA is installed, try launching it from the operating system. It should open in a standalone window without an address bar and with the PWA theme color.
+
+#### Use the app without connection
```

Comment:
For offline, I think its important to mention that usually you'd have to have loaded the site at least once before offline stuff would work. Bc it cache's the data and yeah